src/pad_action_entry.rs

```diff
@@ -4,50 +4,53 @@
 
 use glib::translate::*;
 use gtk_sys;
+use std::ffi::CStr;
 use PadActionType;
 
-#[repr(C)]
-pub struct PadActionEntry(gtk_sys::GtkPadActionEntry);
+#[derive(Debug, Clone)]
+pub struct PadActionEntry {
+    pub(crate) type_: PadActionType,
+    pub(crate) index: i32,
+    pub(crate) mode: i32,
+    pub(crate) label: String,
+    pub(crate) action_name: String,
```

Comment:
Actually... as we have getters for all these, they can as well stay completely private and you can use the getters in the other file :)